ocs_ci/utility/prometheus.py

```
@@ -881,6 +881,32 @@ def wait_for_alert(self, name, state=None, timeout=1200, sleep=5, min_count=1):
+                (e.g. {"source_bucket": "my-bucket"}).
+
+        Returns:
+            list: Matching alert records, empty if none found.
+        """
+        with self._cluster_context():
+            response = self.get(
+                "alerts",
+                payload={"silenced": False, "inhibited": False},
+            )
+            if not response.ok:
+                raise AlertingError(f"Request {response.request.url} failed")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)